ludwig/schema/metadata/trainer.py

```diff
@@ -0,0 +1,594 @@
+batch_size_metadata = {
+    "allow_none": False,
+    "default_value": 128.0,
+    "ui_display_name": "Batch Size",
+    "type_annotation": "int",
+    "description": "The number of training examples utilized in one training step of the model. If ’auto’, we will use the biggest batch size (power of 2) that can fit in memory.",  # noqa: E501
+    "default_value_reasoning": "Not too big, not too small.",
+    "schema": 'min: 1, or "auto"',
```

Comment:
To avoid duplication and unintentional override conflicts, it would probably make sense to exclude the fields that are used to select the schema in the first place (`type_annotation`, `schema`), and the fields that are used to initialize the schema (`allow_none`, `default_value`, `description`, `schema`). WDYT?